### PR TITLE
Regen bindings

### PR DESCRIPTION
**Pull request by Manishearth**
(based on #14330)

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/14354)

### CODE REVIEW

File: components/style/binding_tools/regen.py [hunk 1]
```diff
@@ -328,9 +328,15 @@
             "nsStyleVisibility",
             "nsStyleXUL",
         ],
+        "array_types": {
+            "uintptr_t": "usize",
```
Comment (emilio) [inline components/style/binding_tools/regen.py]: Such an abuse... :(

**Review by emilio — APPROVED** [review]
r=me if needed, though check w/ @bholley, since this is going to bitrot #14300

File: components/style/binding_tools/regen.py (hunk 1 above)
Comment (Manishearth) [inline components/style/binding_tools/regen.py]: I didnt make it :)

**Comment by emilio** [comment]
This is now stale as of https://github.com/servo/servo/pull/14457, so I'm closing this. Let me know if something here needs landing though.

**Pull request closed without merging**